### PR TITLE
SL-4613 Extend post object to contain image details and helper methods

### PR DESCRIPTION
**Pull request by sergioebx**
### Description of Changes
- Add images map to `Post` object
- Add helper methods to retrieve `title` and `description`

## Final Checklist

Please tick once completed.

- [ ] Build passes.
- [ ] Versioning considered (the version number in this PR is inline with semantic 
versioning requirements).
- [ ] Change log has been updated.

### CODE REVIEW

File: src/main/java/com/echobox/api/linkedin/types/posts/Post.java [hunk 1]
```diff
@@ -189,6 +195,51 @@ public class Post extends LinkedInURNIdType {
...
+    Content content = this.getContent();
+    if (content == null) {
+      return null;
+    }
+    
+    if (content.getArticle() == null) {
+      return null;
+    }
+    
+    return content.getArticle().getDescription();
```
Comment (kennethwong-hc) [inline src/main/java/com/echobox/api/linkedin/types/posts/Post.java:233]:
```suggestion
    return Optional.ofNullable(this.getContent())
        .map(Content::getArticle)
        .map(ArticleContent::getDescription)
        .orElse(null);
```

File: src/main/java/com/echobox/api/linkedin/types/posts/Post.java [hunk 2]
```diff
@@ -189,6 +195,51 @@ public class Post extends LinkedInURNIdType {
...
+  }
+  
+  public List<String> getImageURLs() {
+    return images.values().stream()
+        .map(ImageDetails::getDownloadUrl)
+        .filter(StringUtils::isNoneBlank)
```
Comment (kennethwong-hc) [inline src/main/java/com/echobox/api/linkedin/types/posts/Post.java:239]:
```suggestion
        .filter(StringUtils::isNotBlank)
```
Why don't we just use isNotBlank? as downloadUrl is a simple String only

**Review by kennethwong-hc — COMMENTED** [review]
Looks good. Please see my comments.

File: src/main/java/com/echobox/api/linkedin/types/posts/Post.java (hunk 2 above)
Comment (sergioebx) [reply src/main/java/com/echobox/api/linkedin/types/posts/Post.java]: done

**Review by kennethwong-hc — APPROVED** [review]
Great! Thanks!

**Pull request merged**